### DISCUSSION TITLE
好书与课程推荐

### DISCUSSION THREAD

**Discussion by chenxiaoyao6228** [cuixiaorui/mini-vue#60] [Ideas]
分享大家看过的书或者课

**Comment by chenxiaoyao6228** [comment]
课程: <浏览器工作原理与实践>
理由: 看完能对浏览器的运行机制有更好的理解

**Comment by Stan9726** [comment]
- 课程：<浙江大学数据结构（陈越&何钦铭）>
- 理由：老师讲课通俗易懂、由浅入深，适合系统入门数据结构与算法
- 地址：[中国大学MOOC_数据结构](https://www.icourse163.org/course/ZJU-93001)或者[B站_数据结构-浙江大学](https://www.bilibili.com/video/BV1JW411i731?share_source=copy_web)
- 其他：免费，在中国大学MOOC上报名还可以获得作业和参加考试，结束后可以获得证书

**Comment by alexzhang1030** [comment]
- 课程：<10分钟快速入门计算机科学>
- 理由：每一节 10 分钟（目前共 40 集），帮助你快速了解计算机相关的专业知识
- 地址：https://www.bilibili.com/video/av21376839/
- 其他：[Github 地址](https://github.com/1c7/Crash-Course-Computer-Science-Chinese)
